build-requirements.pip

```diff
@@ -1,2 +1,3 @@
 gitversion
 beautifulsoup4
+six
```

Comment:
This shouldn't be necessary - `build-requirements.pip` isn't even used on Travis at the moment. (And I think I need to actually remove beautifulsoup4 too, not sure when that crept in.)